frontend-instructor/src/components/mycourses/MyCourses.jsx

```
@@ -1,15 +1,95 @@
+                  <p className="text-yellow-500">Rating: {course.rating ?? "N/A"} ⭐</p>
+                  <p className="text-lg font-bold text-blue-500">৳ {course.price}</p>
 
-export default MyCourses;
+                  <div className="mt-2 flex gap-3">
+                    <Link to={`/courses/${course.id}`}>
+                      <button className="px-4 bg-blue-500 text-white py-2 rounded-lg hover:bg-blue-600 transition">
+                        View Details
+                      </button>
+                    </Link>
+                    <button className="px-4 bg-gray-500 text-white py-2 rounded-lg hover:bg-gray-600 transition">
+                      Edit
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)